### ISSUE TITLE
[Accessibility][Carousel][CSS] Currently unselected carousel buttons have insufficient contrast.

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1785]
**Steps to Reproduce**
1. Open page in Google Chrome.
2. View gray carousel controls for slides that aren't currently displayed.
3. Inspect contrast using a contrast tool such as the [Accessible Color Picker](https://chrome.google.com/webstore/detail/accessible-color-picker/bgfhbflmeekopanooidljpnmnljdihld?hl=en-US).

**Current Behavior**
The buttons for slides that are not currently selected are gray (#E1E1E1) on a white background. The color contrast ratio is 1.31:1, which does not meet the minimum 3:1 contrast ratio. Users with low vision may have trouble perceiving these buttons.

**Expected behavior/code**
All interactive controls/components must meet a minimum contrast ratio of 3:1. 

**Environment**
Chrome 92, Windows 10
AEM Core Components v. 2.17  

**Affected Users**
Users with low vision

**WCAG SC**
1.4.11 Non-Text Contrast

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES

**Comment by gabrielwalt** [comment]
The best solution would be to not rely on different colors for the different states of the carousel indicator:
* For both states, use the same strongly contrasted grey.
* For the active state, display an additional ring around the dot.
